egs/himia/wuw/README.md

```diff
@@ -0,0 +1,10 @@
+# Pretrained models and related logs/results.
+
+## ctc tdnn baseline
+
+AUC results for different epochs could be found at <https://huggingface.co/GuoLiyong/himia_ctc_tdnn_baseline/tree/main>
+
+E.g. for epoch 15 and avg 1, result log file is: <https://huggingface.co/GuoLiyong/himia_ctc_tdnn_baseline/blob/main/exp_max_duration_100/post/epoch_15-avg_1/log/log-auc-himia_aishell-2023-03-16-17-42-14>
+
+Corresponding ROC curve is: <https://huggingface.co/GuoLiyong/himia_ctc_tdnn_baseline/blob/main/exp_max_duration_100/post/epoch_15-avg_1/himia_aishell.pdf>
```

Comment:
Could you create a RESULTS.md and put some images of ROC curves in it for ease of reference.

Reply:
Good suggestions. Added. Thanks.